requirements/tox.txt

```diff
@@ -0,0 +1,24 @@
+# edX Internal Requirements
+git+https://github.com/edx/XBlock.git@xblock-0.4.12#egg=XBlock==0.4.12
+git+https://github.com/edx/xblock-sdk.git@v0.1.4#egg=xblock-sdk==0.1.4
+
+# edx-submissions
+git+https://github.com/edx/edx-submissions.git@2.0.1#egg=edx-submissions==2.0.1
+
+# Third Party Requirements
+boto>=2.32.1,<3.0.0
+python-swiftclient==3.1.0
+defusedxml==0.5.0
+django-model-utils==3.0.0
+django-nose==1.4.4
+git+https://github.com/edx/django-rest-framework.git@1ceda7c086fddffd1c440cc86856441bbf0bd9cb#egg=djangorestframework==3.6.3
+dogapi==1.11.1
+jsonfield==2.0.2
```

Comment:
@MichaelRoytman just saw your recent branch - I'm upgrading `jsonfield` here as part of https://openedx.atlassian.net/browse/EDUCATOR-768